appshell/cefclient.cpp

```diff
@@ -95,7 +98,19 @@ void AppGetSettings(CefSettings& settings, CefRefPtr<CefCommandLine> command_lin
       command_line->GetSwitchValue(client::switches::kJavascriptFlags);
     
   // Enable dev tools
-  settings.remote_debugging_port = REMOTE_DEBUGGING_PORT;
+  CefString debugger_port = command_line->GetSwitchValue("remote-debugging-port");
+  if (!debugger_port.empty()) {
+    int port = atoi(debugger_port.ToString().c_str());
+    static const int max_port_num = static_cast<int>(std::numeric_limits<uint16_t>::max());
```

Comment:
Can we get away without needing limits header? 
I am not comfortable with the need for unsetting standard functions.
You can compute uint16 max as such, will work for all platforms
uint16 max_limit = 0;
max_limit = ~max_limit;
max_limit = max_limit >> 1

Reply:
std::numeric_limits<> is C++ equivalent of C macros denoting MAX and MIN of a default numeric data types. So, instead of `std::numeric_limits<uint16_t>::max()`, I could use `UINT16_MAX` defined in [stdint.h](https://en.cppreference.com/w/cpp/header/cstdint)

Reply:
Don't bother about header, it's C++, not JS.